<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>995</width>
    <height>484</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,1,3,4">
    <property name="leftMargin">
     <number>9</number>
    </property>
    <property name="topMargin">
     <number>8</number>
    </property>
    <property name="rightMargin">
     <number>9</number>
    </property>
    <property name="bottomMargin">
     <number>9</number>
    </property>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QWidget" name="enemy_hand_widget" native="true"/>
      </item>
      <item>
       <widget class="QWidget" name="enemy_board_widget" native="true"/>
      </item>
      <item>
       <widget class="QWidget" name="ally_board_widget" native="true"/>
      </item>
      <item>
       <widget class="QWidget" name="ally_hand_widget" native="true"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>120</number>
      </property>
      <item>
       <widget class="EntityWidget" name="enemy_hero_widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>75</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="EntityWidget" name="game_widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>75</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="EntityWidget" name="ally_hero_widget" native="true">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>75</height>
         </size>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QTextBrowser" name="log"/>
    </item>
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QPushButton" name="atk_btn">
          <property name="text">
           <string>Attack</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="atk_from_lbl"/>
        </item>
        <item>
         <widget class="QLineEdit" name="atk_to_lbl"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="play_card_btn">
          <property name="text">
           <string>Play Card</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="play_lbl">
          <property name="inputMask">
           <string/>
          </property>
          <property name="text">
           <string/>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="checkBox">
          <property name="text">
           <string>to left</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="end_turn_btn">
          <property name="text">
           <string>End turn</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>EntityWidget</class>
   <extends>QWidget</extends>
   <header location="global">entitywidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
